dist/profile/manifests/jenkinscontroller.pp

```diff
@@ -430,6 +415,16 @@
 RewriteCond %{REQUEST_FILENAME} ^(.*)people(.*)$ [NC]
 RewriteRule ^.* \"https://jenkins.io/infra/ci-redirects/\"  [L]
 
+# Loading our Proxy rules ourselves from a custom fragment since the
+# puppetlabs/apache module doesn't support ordering of both proxy_pass and
+# proxy_pass_match configurations
+ProxyRequests Off
+ProxyPreserveHost On
+ProxyPass / http://localhost:8080/ nocanon
+ProxyPassReverse / http://localhost:8080/
+"
+  if $block_api_json_path {
+    $custom_fragment = "${custom_fragment}
```

Comment:
Great job! I remember that a variable in Puppet can only be set once and is immutable then so this line seems weird.

Were you able to try this in the vagrant-docker stack? I'm not 100% sure of myself on this so acceptance testing looks the way to go ;)

Reply:
I also saw `unless $groovy_init_enabled {` elsewhere, but I'm not sure of the difference it would make instead of the `if`

Reply:
Added a $base_custom_fragment